site/documentation/content/user-guide/device-registry.md

```diff
@@ -214,278 +216,187 @@ The following command retrieves registration data for device `4711`:
-Please refer to the [Credentials API]({{< relref "/api/Credentials-API.md" >}}) for the specific elements that are explained in detail there.
+Please refer to the [Device Registry HTTP API](/hono/docs/latest/api/device-registry-v1.yaml) for the specific elements that are explained in detail there.
 
-### Add Credentials for a Device
+### Update Credentials for a Device
 
-* URI: `/credentials/${tenantId}`
-* Method: `POST`
+* URI: `/v1/credentials/${tenantId}/${deviceId}`
+* Method: `PUT`
 * Request Headers:
   * (required) `Content-Type`: `application/json` (no other type supported)
 * Request Body:
-  * (required) A JSON object as specified by [Credentials Format]({{< relref "/api/Credentials-API.md#credentials-format" >}}) of the Credentials API.
+  * (required) A JSON object as specified by [Credentials schema] (/hono/docs/latest/api/device-registry-v1.yaml) of the HTTP API specification.
 * Status Codes:
-  * 201 (Created): Credentials have been added successfully under the resource indicated by `Location` header.
+  * 204 (No Content): Credentials have been updated successfully.
   * 400 (Bad Request): The credentials have not been added because the request was malformed, e .g. because the payload did not contain required values. The response body may contain hints regarding the cause of the problem.
-  * 409 (Conflict): Credentials of the given type for the given *auth-id* already exist for the tenant. The request has not been processed.
 * Response Headers:
-  * `Location`: The URI under which the newly created resource can be accessed.
+  * `ETag`: Version of the resource
 
 **Example**
 
 The following command adds some `hashed-password` credentials from a given plain text password for device `4710` using authentication identifier `sensor10`: 
 
-    curl -i -X POST -H 'Content-Type: application/json' --data-binary '{
-      "device-id": "4710",
+    curl -i -X PUT -H 'Content-Type: application/json' --data-binary '[{
       "type": "hashed-password",
       "auth-id": "sensor10",
       "secrets": [{
           "pwd-plain": "mylittlesecret"
       }]
-    }' http://localhost:28080/credentials/DEFAULT_TENANT
+    }]' http://localhost:28080/v1/credentials/DEFAULT_TENANT/4710
       
-    HTTP/1.1 201 Created
-    Location: /credentials/DEFAULT_TENANT/sensor10/hashed-password
-    Content-Length: 0
+    HTTP/1.1 204 No Content
+    ETag: becc93d7-ab0f-48ec-ad26-debdf339cbf4x
     
 This uses a convenient option which lets the Device Registry do the hashing of the password. The following command retrieves the credentials that are stored by the Device Registry as a result of the command above: 
     
     
-    curl -i http://localhost:28080/credentials/DEFAULT_TENANT/4710
-    
+    curl -i http://localhost:28080/v1/credentials/DEFAULT_TENANT/4710
+
     HTTP/1.1 200 OK
     Content-Type: application/json; charset=utf-8
-    Content-Length: 222
 
-    {
-      "total": 1,
-      "credentials": [
+    [{
```

Comment:
IMHO  the response also contains an *etag* header here

Reply:
yup